locales/zh_CN/LC_MESSAGES/taichicon.po

```diff
@@ -3,281 +3,299 @@
 # This file is distributed under the same license as the taichi package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: taichi 0.6.10\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-06-14 12:50+0800\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2020-06-16 22:48+0800\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: zh_CN\n"
+"X-Generator: Poedit 2.3.1\n"
 
 #: ../../taichicon.rst:2
 msgid "TaichiCon"
-msgstr ""
+msgstr "Taichi 开发者讨论会"
 
 #: ../../taichicon.rst:4
 msgid ""
-"We are hosting a series of online **TaichiCon** events for developers to "
-"gather and share their Taichi experiences."
+"We are hosting a series of online **TaichiCon** events for developers to gather "
+"and share their Taichi experiences."
 msgstr ""
+"我们正在举办一系列线上的 **TaichiCon（Taichi 讨论会）** 活动，让开发人员能够聚集"
+"在一起，分享他们使用的 Taichi 经验。"
 
 #: ../../taichicon.rst:7
 msgid "Past Conferences"
-msgstr ""
+msgstr "历届讨论会"
 
 #: ../../taichicon.rst:9
 msgid ""
-"Everything on previous TaichiCons is available in the `TaichiCon "
-"<https://github.com/taichi-dev/taichicon>`_ repository."
+"Everything on previous TaichiCons is available in the `TaichiCon <https://"
+"github.com/taichi-dev/taichicon>`_ repository."
 msgstr ""
+"历届讨论会的所有相关资料都可以在 `TaichiCon <https://github.com/taichi-dev/"
+"taichicon>`_ 仓库中找到."
 
 #: ../../taichicon.rst:12
 msgid "Format"
-msgstr ""
+msgstr "会议流程"
 
 #: ../../taichicon.rst:14
 msgid "Each TaichiCon consists of two parts: **talks** and **free discussions**."
-msgstr ""
+msgstr "目前，每届 TaichiCon 都会包含两个议程： **讲座** 与 **自由讨论**。"
 
 #: ../../taichicon.rst:16
 msgid "The first 1-hour consists of four 10-minute talks, each with 5-minute Q&A;"
-msgstr ""
+msgstr "最初的一个小时包含了4个10分钟的讲座，每个讲座后留有5分钟的问答时间;"
 
 #: ../../taichicon.rst:18
 msgid ""
-"After the formal talks, attendees are free to chat about Taichi in any "
-"aspects."
-msgstr ""
+"After the formal talks, attendees are free to chat about Taichi in any aspects."
+msgstr "在所有正式的讲座结束之后，与会者可以自由讨论 Taichi 相关的各个方面。"
 
 #: ../../taichicon.rst:20
 msgid "The conference format may evolve in the future."
-msgstr ""
+msgstr "讨论会的流程以后也许会有所更动。"
 
 #: ../../taichicon.rst:23
 msgid "Language"
-msgstr ""
+msgstr "会议语言"
 
 #: ../../taichicon.rst:25
 msgid "The ``i``-th TaichiCon will be hosted in"
-msgstr ""
+msgstr "第 ``i`` 届 TaichiCon 的主持语言将会使用"
 
 #: ../../taichicon.rst:27
 msgid "English ``(if i % 2 == 1)``;"
-msgstr ""
+msgstr "英语 ``(if i % 2 == 1)``;"
 
 #: ../../taichicon.rst:28
 msgid "Chinese (Mandarin) ``(if i % 2 == 0)``."
-msgstr ""
+msgstr "汉语（普通话） ``(if i % 2 == 0)``."
```

Comment:
```suggestion
msgstr "汉语（普通话） ``(若 i % 2 == 0)``."
```

Reply:
Btw, I think the `i % 2` here is too ad-hoc... @yuanming-hu May I replace it by `(even number)` and `(odd number)`? If your hope is to make user feel humorous then I'm fine with this approach :)